ui/src/react-admin/modules/content-page/hooks/useGetLanguageFilterOptions.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { useGetAllLanguages } from '~modules/translations/hooks/use-get-all-languages';
+import { CheckboxOption } from '~shared/components/CheckboxDropdownModal/CheckboxDropdownModal';
+import { tText } from '~shared/helpers/translation-functions';
+
+export const useGetLanguageFilterOptions = (): [CheckboxOption[], boolean] => {
+	const [languageOptions, setLanguageOptions] = useState<CheckboxOption[]>([]);
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+
+	const { data: allLanguages, isLoading: isLoadingAllLanguages } = useGetAllLanguages();
+
+	useEffect(() => {
+		if (!allLanguages) {
+			return;
+		}
+
+		setLanguageOptions([
+			...allLanguages.map(
+				(languageInfo): CheckboxOption => ({
+					label: tText('{{language}} bestaat', {
```

Comment:
This won't be picked up correctly by the extract script, use a [dictionary pattern](https://github.com/viaacode/avo2-client/blob/release/1.27/src/shared/components/BlockList/blocks/AssignmentBlockTypeSearch.const.ts#L6).

Reply:
It should be picked up correctly. We use it often:
![image](https://github.com/viaacode/react-admin-core-module/assets/1710840/55707391-8b7f-45ae-856e-8d7880f4685e)



You can't use variable concatination eg:
```
tText(language + ' bestaat');
```

But you can use parameters in the translation. eg:
```
tText('{{language}} bestaat', { language });
```

But you cannot use parentheses in the variables:
```
tText('{{language}} bestaat', { language: getLanguage() });
```